packages/auth-construct/src/types.ts

```diff
@@ -199,11 +199,19 @@ export type ExternalProviderOptions = {
   /**
    * List of allowed redirect URLs for the identity providers.
    */
-  callbackUrls?: string[];
+  callbackUrls: string[];
```

Comment:
not strictly part of this PR, but this should either be `redirectUrls` to match the comment, or change the comment to say "callback URLs"

Reply:
![image](https://github.com/aws-amplify/amplify-backend/assets/110861985/04b60250-f9fc-4f1a-862e-8f17a479298f)
I'm not sure why but CDK uses the names interchangeably here; I'm not sure if this is intentional